config/servers.yml

```diff
@@ -91,7 +91,6 @@
 #        concurrency:               1
 #        requiredAcks:              1
 #        compressionCodec:          none
-#        autoCommitEnabled:         true
```

Comment:
Yes,  we should allow for disabling `autoCommitEnabled` at consumer level in a future release, but with the only option available right now to set it globally, it's a very cumbersome setting for the end user. It also requires thorough documentation, including a description of the acknowledgment mechanism in Spring Integration Kafka. 

Captured as a requirement as part of https://jira.spring.io/browse/XD-3164.